src/worker.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
I don't think this matters, but I think the file is called service-worker.js in the template boilerplate. Something to remember, maybe?

Reply:
👍